### ISSUE TITLE
/Ghost command is inconsistent on its effect

### ISSUE THREAD

**Issue by Titian3** [space-wizards/space-station-14#24668]
## Description
When using the /ghost command if you are in crit it is a reversible operation but if you use it normally it is not and is essentially /suicide.
Some warning of what will happen or usage information should be given about it to players so they don't accidentally remove them self from the round.
Based off it seen actually happening during admin'ing.

**Reproduction**
- do /Ghost whilst alive, never return.
- do /Ghost whilst crit, able to return.

**Screenshots**
Post /ghost whilst still alive.
![image](https://github.com/space-wizards/space-station-14/assets/47093363/a4274ec5-e5f1-4138-ac4a-ede33921e4ab)

**Additional context**
NA

**Comment by lzk228** [comment]
intended?

**Comment by LordEclipse** [comment]
It is intentional, however, it should be more obvious or visible on its effect.

**Comment by deltanedas** [comment]
now that we have crit actions /ghost can be made consistent

**Comment by Titian3** [comment]
> now that we have crit actions /ghost can be made consistent

This is true, potentially /ghost doesn't even need to exist due to that functionality being available in those actions.

**Comment by deltanedas** [comment]
no its still good if you are leaving so people know you wont return

**Comment by Titian3** [comment]
> no its still good if you are leaving so people know you wont return

/suicide command

**Comment by deltanedas** [comment]
nrp

**Comment by LordEclipse** [comment]
> > no its still good if you are leaving so people know you wont return
> 
> /suicide command

/ghost is important for ghost roles. If you /suicide then that role is just gone for no reason if they don't want to use it. Also some station pets are literally imposisble to /suicide so people would be effectively trapped in them.
